draft-ietf-quic-http.md

```diff
@@ -261,16 +261,23 @@ the stream management.
 All client-initiated bidirectional streams are used for HTTP requests and
 responses.  A bidirectional stream ensures that the response can be readily
 correlated with the request. This means that the client's first request occurs
-on QUIC stream 0, with subsequent requests on stream 4, 8, and so on. HTTP/QUIC
-does not use server-initiated bidirectional streams. The use of unidirectional
-streams is discussed in {{unidirectional-streams}}.
+on QUIC stream 0, with subsequent requests on stream 4, 8, and so on. In order
+to permit these streams to open, an HTTP/QUIC client SHOULD send non-zero values
+for the QUIC transport parameters `initial_max_stream_data_bidi_local`. An
+HTTP/QUIC server SHOULD send non-zero values for the QUIC transport parameters
+`initial_max_stream_data_bidi_remote` and `initial_max_bidi_streams`.
```

Comment:
Would it make sense to recommend `initial_max_bidi_streams` to be "no smaller than 100", like what RFC 7540 states?

Reply:
As I don't really follow HTTP/Q atm., I am not very confident in this - but if QPACK uses static tables only and you are on a constrained device, you would not want to track that much state.

Reply:
I do not oppose to your point.

What I am looking for is a general guidance that is similar to what we have in RFC 7540, quote: _”It is recommended that this value be no smaller than 100, so as to not unnecessarily limit parallelism.”_